go.mod

```diff
@@ -122,6 +122,7 @@ require (
 	google.golang.org/grpc v1.45.0
 	gopkg.in/yaml.v2 v2.4.0
 	honnef.co/go/tools v0.3.3
+	k8s.io/apimachinery v0.26.0
```

Comment:
Where is this used?

Reply:
for parsing the unit: https://github.com/pingcap/tidb/pull/40188/files#diff-f76f1dfea4a7a14d9125895ab90a9720a1819836d67701ca082ce93c4b11f981R62

I may replace it with own implement after this PR.